builds/checkin/mqtt.yaml

```diff
@@ -0,0 +1,81 @@
+trigger: none
+pr:
+  branches:
+    include:
+      - master
+      - release/*
+jobs:
+
+  ################################################################################
+  - job: check_run_pipeline
+    ################################################################################
+    displayName: Check pipeline preconditions (changes ARE in builds or mqtt)
+    pool:
+      vmImage: "ubuntu-16.04"
+    steps:
+      - bash: |
+          git log -m -1 --name-only --first-parent --pretty="" | egrep -i '^(builds|mqtt)'
+          if [[ $? == 0 ]]; then
+            echo "Detected changes inside builds or mqtt folders"
+            echo "##vso[task.setvariable variable=RUN_PIPELINE;isOutput=true]TRUE"
+          fi
+        displayName: Check changes in sources
+        name: check_files
+
+  ################################################################################
+  - job: linux_amd64
```

Comment:
will cover ARM32 and ARM64 in next PR?

Reply:
I tried to simplify this pipeline. Eventually we add support for arm based ones as well. Not now